block/src/Pos/Block/Configuration.hs

```diff
@@ -47,6 +48,8 @@ data BlockConfiguration = BlockConfiguration
       -- ^ Estimated time for broadcasting messages
     , ccRecoveryHeadersMessage :: !Int
       -- ^ Numbers of headers put in message in recovery mode.
+    , ccStreamWindow           :: !Int
```

Comment:
Is it necessary that client and server agree on this? That's the case for `ccRecoveryHeadersMessage` unfortunately.

Reply:
No, the server will follow the clients ccStreamWinndow.